example/echo-contract.js

```diff
@@ -57,5 +84,32 @@ const echoContract = async (ctx) => {
     // await ctx.updateConfig(config);
 }
 
+const fallback = async (ctx) => {
+    // NPL messages example.
+    // Start listening to incoming NPL messages before we send ours.
+    const promise = new Promise((resolve, reject) => {
+        let timeout = setTimeout(() => {
+            reject('NPL timeout.');
+        }, 2000);
+
+        let list = [];
+        ctx.unl.onMessage((node, msg) => {
+            console.log(`${node.publicKey} said ${msg} to me.`);
+            list.push(msg);
+            if (list.length == ctx.unl.list().length) {
+                clearTimeout(timeout);
+                resolve();
+            }
+        });
+    });
+
+    await ctx.unl.send("Hello");
+    await promise;
+}
+
 const hpc = new HotPocket.Contract();
-hpc.init(echoContract);
\ No newline at end of file
+hpc.init({
+    "consensus": contract,
+    "consensus_fallback": fallback,
+    "read_req": async (ctx) => { await contract(ctx, true) }
+});
```

Comment:
Same example I have given in elsewhere. I think following coding pattern makes it easier to follow since the function call syntax is more consistent.

```
hpc.init({
    "consensus": async (ctx) => { await contract(ctx, false); },
    "read_req":  async (ctx) => { await contract(ctx, true); },
    "consensus_fallback":  async (ctx) => { await fallback(ctx); }
});
```